addons/rose/addon/styles/rose/components/form/_radio-card.scss

```diff
@@ -0,0 +1,219 @@
+@use '../../variables/sizing';
+@use '../../utilities/type';
+@use '../../variables/typography';
+
+$l: sizing.rems(l); // 16
+$m: sizing.rems(m); // 16
+$size-xs: sizing.rems(xs); // 8
+$size-xxs: sizing.rems(xxs); // 4
+$size-xxxs: sizing.rems(xxxs); // 2
+$size-xxxxs: sizing.rems(xxxxs); // 1
+
+.rose-form-radio-card-wide {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: ($m) * 23;
+  justify-content: center;
+
+  .radio-card-overlay {
+    position: absolute;
+    height: $l + $size-xxs;
+    background-color: var(--ui-gray-subtler-5);
+    width: 100%;
+    bottom: 0;
+    right: 0;
+  }
+  .rose-form-radio-label {
+    font-style: normal;
+    font-weight: bold;
+    font-size: typography.font-size(s);
+    color: var(--black);
+    width: ($m) * 23;
+    height: ($m) * 7.5;
+    display: block;
+    position: relative;
+    text-align: left;
+    word-break: break-all;
+    border: $size-xxxxs solid #bac1cc;
+    box-shadow: 0 $size-xxs $size-xxs
+      rgba(var(--stark-components), var(--opacity-2));
+    border-radius: $size-xxxs;
+    padding: $s $l;
+    box-sizing: border-box;
+    background-color: var(--white);
+    .rose-form-helper-text {
+      font-weight: normal;
+    }
+    .rose-icon {
+      padding-right: $size-xxs;
+      padding-left: $size-xxs;
+    }
+  }
+  .rose-form-radio-field {
+    position: absolute;
+    z-index: 1;
+    bottom: $size-xs;
+    &:not(:disabled) {
+      &:hover {
+        ~ .rose-form-radio-label {
+          border: $size-xxxxs solid var(--ui-gray);
+        }
+      }
+      &:checked {
+        ~ .rose-form-radio-label {
+          .radio-card-overlay {
+            background-color: var(--action-subtler-1h);
+          }
+          border: $size-xxxxs solid #1563ff;
```

Comment:
Unless there is a specific reason, we should use the colors in the styleguide with its variable.